### PR TITLE
Update PY2 in docs to PY3; #78

### LINKED ISSUE #78: Update PY2 in docs to PY3
Site runs on Python 3 now, so all instances of Python 2 (e.g. `python-dev` vs `python3-dev`) should be replaced with their Python 3 counterparts.

### PR DESCRIPTION
**Pull request by Ninjaclasher**
Closes #78.

### CODE REVIEW

**Comment by kiritofeng** [comment]
Doesn't close #78, as the judge setup docs still use PY2, but otherwise LGTM

**Comment by Ninjaclasher** [comment]
All python 2 instances should be updated now (I think).

File: docs/site/installation.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 ## Installing the prerequisites
 
 ```shell-session
-$ apt install git gcc g++ make python-dev libxml2-dev libxslt1-dev zlib1g-dev gettext curl wget redis-server
-$ wget -q --no-check-certificate -O- https://bootstrap.pypa.io/get-pip.py | sudo python
-$ pip install virtualenv
+$ apt update
+$ apt install git gcc g++ make python3-dev libxml2-dev libxslt1-dev zlib1g-dev gettext curl wget redis-server
+$ wget -q --no-check-certificate -O- https://bootstrap.pypa.io/get-pip.py | sudo python3
```
Comment (kiritofeng) [inline docs/site/installation.md:6]: Can you change this to install `python3-pip` from `apt` instead?

Comment (Ninjaclasher) [reply docs/site/installation.md]: Done

File: docs/site/installation.md [hunk 2]
```diff
@@ -1,9 +1,8 @@
 ## Installing the prerequisites
 
 ```shell-session
-$ apt install git gcc g++ make python-dev libxml2-dev libxslt1-dev zlib1g-dev gettext curl wget redis-server
-$ wget -q --no-check-certificate -O- https://bootstrap.pypa.io/get-pip.py | sudo python
-$ pip install virtualenv
+$ apt update
+$ apt install git gcc g++ make python3-dev python3-pip libxml2-dev libxslt1-dev zlib1g-dev gettext curl wget redis-server
```
Comment (kiritofeng) [inline docs/site/installation.md:5]: You no longer need `wget`

**Pull request merged**